library/x509_crl.c

```diff
@@ -260,6 +281,9 @@ int mbedtls_x509_crl_parse_der( mbedtls_x509_crl *chain,
     unsigned char *p, *end;
     mbedtls_x509_buf sig_params1, sig_params2, sig_oid2;
     mbedtls_x509_crl *crl = chain;
+    mbedtls_x509_crl_entry *entry;
+    int i, count;
```

Comment:
Its better to just rename it. re-using crl->entries_num might create confusion and make the code more complex.